proto/v2/instance.go

```diff
@@ -0,0 +1,15 @@
+package proto
+
+import "time"
+
+type Instance struct {
+	Subsystem
+	ParentUUID string
+	UUID       string
+	Name       string
+	DSN        string
+	Created    time.Time
+	Deleted    time.Time
+	Properties map[string]string
+	Subsystems []Instance
+}
```

Comment:
Let's group "high cohesive" structs in the same file. Right now, I have to `vi instance.go subsystem.go system_tree_sync.go` to get a complete picture.